hs_core/tests/api/rest/test_content_types.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+from rest_framework.test import APIClient
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+
+class TestContentTypes(APITestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+        self.content_types = {'GenericLogicalFile',
+                               'GeoRasterLogicalFile',
+                               'NetCDFLogicalFile',
+                               'GeoFeatureLogicalFile',
+                               'RefTimeseriesLogicalFile',
+                               'TimeSeriesLogicalFile',
+                               'FileSetLogicalFile'}
```

Comment:
I actually prefer to have these hard coded in the tests, it creates redundancy.  Yes, if we add a logicalFile in the future, the person adding it will have to update this test, but I think it's a good thing.